src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java

```diff
@@ -11,8 +11,9 @@
 
 public class ChatTestsStaff extends NewTestTemplate {
 
-  private String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
-  private String MESSAGE_ON_PRIVATE_CHAT = "Test message on private chat";
+  private final String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1170)